ext/tokenizer/Makefile.frag

```diff
@@ -1,4 +1,7 @@
 $(top_srcdir)/Zend/zend_language_parser.c:
 $(top_srcdir)/Zend/zend_language_scanner.c:
-$(top_srcdir)/ext/tokenizer/tokenizer_data.c: $(top_srcdir)/Zend/zend_language_parser.h
+$(top_srcdir)/ext/tokenizer/tokenizer_data.c: $(top_srcdir)/Zend/zend_language_parser.h $(top_srcdir)/Zend/zend_language_parser.y
```

Comment:
Not sure if `zend_language_parser.h` can be removed.

Reply:
Looks like it's needed, otherwise the script will be called before parser.h updated.